adapters/postgres/postgres.go

```diff
@@ -353,36 +413,15 @@ func parseArray(value interface{}) string {
 }
 
 // Insert execute insert sql into a table
-func Insert(database, schema, table string, body api.Request) (jsonData []byte, err error) {
+func Insert(database, schema, table string, r *http.Request) (jsonData []byte, err error) {
```

Comment:
@marioidival  i'll do it in next PR when will solve https://github.com/nuveo/prest/issues/142